assets/README.md

```diff
@@ -0,0 +1,36 @@
+# Mayflower-Tokens
+[![npm package][npm-badge]][npm]
+
+Mayflower-tokens provides a bundled up library of design tokens and assets that you will need for theming your front end framework to look and feel like [Mass.gov](https://mass.gov) and other Commonwealth web properties that are in Mayflower ecosystem. Mayflower-tokens includes color token SCSS variables, fonts, icons and other imagery from Mayflower — the [enterprise design system for the Commonwealth of Massachusetts][mayflower-doc].
+
+- **Codebase:** [Mayflower monorepo][mayflower-github] `assets` subdirectory
+  >[Mayflower monorepo][mayflower-github] comprised of [Mayflower documentation][mayflower-doc], two component libraries — [Mayflower React][react-storybook] and [Mayflower PatternLab][patternLab], and their [shared assets][shared-assets].
+  > Refer to [Mayflower PatternLab Static Site][patternlab] for the set of UI components consumed in Mass.gov.
+- **Live demo:** [mayflower-react storybook][react-storybook]
+
+## Using Mayflower-Tokens in Your Project
+1. Install mayflower-tokens into your project as a dependency.
+`npm i @massds/mayflower-tokens --save`
+2. Import color tokens into your SASS/SCSS file
+`@import '[path to node_modules]/@massds/mayflower-tokens/colors/mayflower-color_tokens';`
+3. Refer to variables in `@massds/mayflower-tokens/colors/_mayflower-color_tokens.scss` to map the Mayflower color tokens in your theming SCSS.
```

Comment:
Is there a documentation page that could be linked to that provides a visual of what colors these hex values align to and their Sass variable equivalent?

Reply:
yes, in storybook --> https://mayflower-react.digital.mass.gov/?path=/story/brand-colors--colors, will add a link